okhttp/src/jvmMain/kotlin/okhttp3/OkHttpClient.kt

```diff
@@ -227,7 +227,8 @@ open class OkHttpClient internal constructor(
   @get:JvmName("minWebSocketMessageToCompress")
   val minWebSocketMessageToCompress: Long = builder.minWebSocketMessageToCompress
 
-  val routeDatabase: RouteDatabase = builder.routeDatabase ?: RouteDatabase()
+  internal val routeDatabase: RouteDatabase = builder.routeDatabase ?: RouteDatabase()
```

Comment:
This is the behavior change. This should never have been non-internal; it was a mistake to have an `internal` type in a public API.